src/app/ApplicationTemplate.Shared.Views/App.xaml

```diff
@@ -12,9 +12,9 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
```

Comment:
Add screenshots of before vs after in the PR description.

Reply:
but they will look the same, because i left the same colors... but in case we need to add it anyways, which pages?? or would it be better a video instead??

Reply:
A video works :)
I'm assuming that there would be some small changes because it's not the same system?

Reply:
videos created